src/universal/modules/welcome/components/WelcomeWizardForms/index.js

```diff
@@ -0,0 +1,3 @@
+export Step1PreferredName from './Step1PreferredName';
```

Comment:
I put these modules all in the same folder since they are interdependent on the same `redux-form`.